lib/cli-helpers.js

```diff
@@ -10,10 +10,15 @@ const semver = require('semver');
 const namespaceRule = /^[a-zA-Z0-9]+-?[a-zA-Z0-9]+$/;
 const isValidNamespace = (namespace) => namespaceRule.test(namespace);
 
+// The Module name rule is
+// alphanumeric characters or single underscores, and cannot begin or end with a hyphen
+const moduleNameRule = /^[0-9A-Za-z](?:[0-9A-Za-z-_]{0,62}[0-9A-Za-z])?$/;
```

Comment:
The regex has one issue: It allows continuous underscore/hypens, which should not be the case. A better regex should be `^[a-zA-Z0-9]+([-?_?][a-zA-Z0-9]+)+$`

Reply:
I do like this regex better, even though continuous _/- in a name won't break terraform, I don't see there being a need for it so might as well be more opinionated.